src/messages/test_proposals.rs

```diff
@@ -0,0 +1,132 @@
+/// This test makes sure ProposalQueue works as intented. This functionality is used in `create_commit` to filter the epoch proposals.
+/// Expected result:
+/// `get_filtered_proposals` returns only proposals of a certain type
+#[test]
+fn proposal_queue_functions() {
+    use crate::ciphersuite::*;
```

Comment:
The imports should probably go into the file itself in case there are more functions.